Sources/Commands/SwiftTestCommand.swift

```diff
@@ -966,6 +966,9 @@ final class TestRunner {
             args.append(runnerPath.pathString)
             args.append(contentsOf: runner.extraCLIOptions)
             args.append(testPath.relative(to: localFileSystem.currentWorkingDirectory!).pathString)
+            if runner.path?.components.last == "wasmkit" {
```

Comment:
Do we have a more robust way to check for WasmKit?

Reply:
I don't think so. Yes, it feels somewhat icky to check for a path component, but test runners from toolsets provide no other identification. Given that's all we have, why wouldn't we consider this simple and robust enough?